.github/workflows/test-code.yml

```
@@ -71,11 +71,16 @@ jobs:
         if: ${{ startsWith(matrix.os, 'windows') }}
         working-directory: src
         run: dotnet test --no-restore --no-build
-      - name: Test AOT compilation
+      - name: Test code-first AOT compilation
         working-directory: src/GraphQL.AotCompilationSample
```

```suggestion
        working-directory: src/GraphQL.AotCompilationSample.CodeFirst
```

In PR for target branch I think.

---

moved folder as requested.